README.md

```diff
@@ -4,3 +4,80 @@ This is a WIP implementaton of a MetalLB operator, implementing the [operator pa
 for deploying MetalLB on a kubernetes cluster, as described in the [related design proposal](https://github.com/metallb/metallb/blob/main/design/metallb-operator.md).
 
 ## Note that this is still work in progress and not ready for production by any means
+
+
+## Installation
+
+To install the Metallb Operator using a prebuilt image, run: 
+
+```shell
+make deploy
+```
+
+## Usage
+
+Once the Metallb Operator is installed, you have to create a `metallb` custom resource to install Metallb. The operator will consume this resource, and create all required Metallb resources based on it. The `metallb` custom resource needs to be created inside the `metallb-system` namespace and be named `metallb`. Only one `metallb` resource can exist in a cluster.
+
+Below you can find an example of a `metallb` resource definition:
+
+```yaml
+apiVersion: metallb.io/v1alpha1
+kind: Metallb
+metadata:
+  name: metallb
+  namespace: metallb-system
+```
+
+
+## Setting up a development environment
+
+### Running a kind cluster
```

Comment:
Let's try to condense this in what the CI is doing.
What we want to say here is how to build the image locally for a quick dev cycle.

The steps are 

    - name: Build image
      run: |
        docker build . -t ${built_image}

    - name: Create K8s Kind Cluster
      run: |
        kind create cluster --image kindest/node:v1.19.7
        kind load docker-image ${built_image}

    - name: Deploy Metal LB Operator
      run: |
        IMG=${built_image} KUSTOMIZE_DEPLOY_DIR="config/kind-ci/" make deploy

where the initial step can be replaced by the corresponding `make docker-build ` .

Reply:
Yeah I like to see Build changed to 
`IMG=${built_image} make docker-build` instead of using docker directly

Reply:
will be nice of fix it in the template and here but not urgent

Reply:
using make docker-build in the doc
the workflow fix is beyond this pr